profiles/logical/hpos/default.nix

```
@@ -245,6 +245,11 @@ in
   systemd.services.acme-default.serviceConfig.ExecStart =
     lib.mkForce "${holo-router-acme}/bin/holo-router-acme";
 
+  # FIXME: see https://github.com/NixOS/nixpkgs/issues/31540
+  systemd.services.systemd-timesyncd.preStart = ''
+    rm -rf /var/lib/{private/,}systemd/timesyncd
```

```suggestion
    rm -rf /var/lib/{private/,}systemd/timesync
```
according to [issue 31540](https://github.com/NixOS/nixpkgs/issues/31540) this line should read `timesync`. Also I checked in HPOS the name of the folder created by timesyncd is `timesync`.

Unfortunatelly I do not have the way to reproduce issue #460 on my HPOS. Happy to merge and see if it changes anything.